contrib/soju/template.py

```diff
@@ -0,0 +1,60 @@
+pkgname = "soju"
+pkgver = "0.7.0"
+pkgrel = 0
+build_style = "go"
+make_build_args = [
+    "-ldflags",
+    "-X 'git.sr.ht/~emersion/soju/config.DefaultPath=/etc/config/soju' -X 'git.sr.ht/~emersion/soju/config.DefaultUnixAdminPath=/run/soju/admin'",
```

Comment:
```suggestion
    "-ldflags="
    " -X git.sr.ht/~emersion/soju/config.DefaultPath=/etc/config/soju"
    " -X git.sr.ht/~emersion/soju/config.DefaultUnixAdminPath=/run/soju/admin",
```

python joins adjacent strings with no separator so you can use that to split things over lines even when they normally can't be

Reply:
adds a space after ldflags which seems to confuse the rest of the cli parsing
`flag provided but not defined: -X 'git.sr.ht/~emersion/soju/config.DefaultUnixAdminPath
`

Reply:
i already tested exactly what i wrote and it works

Reply:
probably, you are missing the `=` after ldflags